apps/tenant-management-webapp/src/app/pages/admin/services/pdf/templates/templates.tsx

```diff
@@ -122,10 +122,6 @@ export const PdfTemplates: FunctionComponent<PdfTemplatesProps> = ({ openAddTemp
       setFooterPreview(
         generateMessage(template, { data: currentTemplate, serviceUrl: webappUrl, today: new Date().toDateString() })
       );
-      if (currentTemplate) {
-        currentTemplate.footer = footer;
-      }
-      setCurrentTemplate(currentTemplate);
```

Comment:
removing this loses changes between tabs if you haven't saved it yet

Reply:
Thanks, Jonathan! I am going to see whether I can save it when we click the tab.

Reply:
Hi @jonathanweyermann, thank you for the review. I've updated the PR and a copy of the template state will be updated when we click the tab. The update will resolve all of the issues here. Please help me review it again. Thanks!